### ISSUE TITLE
Error: arbitrary-autostart-access. Unable to publish an update

### ISSUE THREAD

**Issue by mijorus** [flathub/flatpak-builder-lint#62]
My app pre-loads itself in memory on startup.

This increases performance on some older devices.
How can I fix this issue?

App: https://github.com/mijorus/smile
```
[view all 16 lines](https://buildbot.flathub.org/#/builders/30/builds/227/steps/6/logs/stdio) 
flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions it.mijorus.smile.json
 in dir /srv/buildbot/worker/build-aarch64-3/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions it.mijorus.smile.json'
 using PTY: False
{
    "errors": [
        "finish-args-arbitrary-autostart-access",
        "finish-args-unnecessary-xdg-config-access"
    ]
}
If you think problems listed above are a false positive, please report it here:
  https://github.com/flathub/flatpak-builder-lint
program finished with exit code 1
elapsedTime=0.459918
```

**Comment by barthalion** [comment]
There's an autostart portal in xdg-desktop-portal. that should be used instead. As for `finish-args-unnecessary-xdg-config-access`, what do you need it for?

**Comment by mijorus** [comment]
could this be the right fix for the issue?

I'm inexperienced with GTK, no idea if I am doing something wrong here

https://github.com/mijorus/smile/commit/ea14de9784b6db2c67dccebfe64bab8efb881651
